pkg/common/dao/models/namespace.go

```diff
@@ -0,0 +1,19 @@
+package models
+
+import (
+	"time"
+
+	"gorm.io/gorm"
+)
+
+// Namespace represents model to work with `namespaces` table.
+type Namespace struct {
+	ID                  uint   `gorm:"primaryKey;autoIncrement"`
+	Code                string `gorm:"unique;index;not null"`
+	Description         string
+	CreatedAt           time.Time
```

Comment:
We have quite a mix in this project, but I think timestamps are usually stored in bigint for the other models (int64 or sql.NullInt64). should we do that here for consistency?

Reply:
that's true. we also have a mix of types for `ids` lie: `uuid`, `string`, `uint`, etc. will be nice to normalise this moment as well.